src/repositories/DocumentSectionRepository.ts

```diff
@@ -1,21 +1,17 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
-import type { DocumentSectionApi, FrontifyDocumentSection } from '../types';
+import type { DocumentSection, DocumentSectionApi } from '../types';
 import { HttpClient } from '../utilities/httpClient';
 
-export const getDocumentSectionsByDocumentPageId = async (
-    documentPageId: number,
-): Promise<FrontifyDocumentSection[]> => {
+export const getDocumentSectionsByDocumentPageId = async (documentPageId: number): Promise<DocumentSection[]> => {
     const { result } = await HttpClient.get<DocumentSectionApi[]>(
         `/api/document-section/?document_page_id=${documentPageId}`,
     );
 
     return result.data.map(mapDocumentSectionApiToFrontifyDocumentSection);
 };
 
-const mapDocumentSectionApiToFrontifyDocumentSection = (
-    documentSectionApi: DocumentSectionApi,
-): FrontifyDocumentSection => ({
+const mapDocumentSectionApiToFrontifyDocumentSection = (documentSectionApi: DocumentSectionApi): DocumentSection => ({
```

Comment:
We should update method name as well:
```suggestion
const mapDocumentSectionApiToDocumentSection = (documentSectionApi: DocumentSectionApi): DocumentSection => ({
```